packages/documentation-framework/components/footer/footer.js

```diff
@@ -137,7 +137,7 @@ export const Footer = () => (
                   <li className="ws-org-pfsite-footer-menu-list-item">
                     <Link
                       className="ws-org-pfsite-footer-menu-link"
-                      to="//slack.patternfly.org"
+                      to="//patternfly.slack.com"
```

Comment:
Suggest using this invite link which (unlike patternfly.slack.com) will allow new users to sign up to join, otherwise will redirect users to the slack workspace if they're already signed in.

```suggestion
                      to="//join.slack.com/t/patternfly/shared_invite/zt-1npmqswgk-bF2R1E2rglV8jz5DNTezMQ"
```